### PR TITLE
New LOC IDs in Tanasee1 (no. 2)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I identified place names in Tanasee1. Because the number is probably too high for one pull request to merge, I push the same records in several new pull requests. Each of these LOC records have been discussed and revised by Dorothea in #94.

### CODE REVIEW

File: new/LOC7393Amyatsma.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LOC7393Amyatsma" xml:lang="en" type="place">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ʾAmʸaṣmā</title>
```
Comment (eu-genia) [inline new/LOC7393Amyatsma.xml:8]: check the symbol - fidel has mʷ

File: new/LOC7393Amyatsma.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        <revisionDesc>
+            <change who="CH" when="2023-10-19">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="area">
+                    <placeName xml:id="n1" xml:lang="gez">አፙጽማ</placeName>
+                    <placeName corresp="#n1" xml:lang="gez">ʾAmʸaṣmā</placeName>
```
Comment (eu-genia) [inline new/LOC7393Amyatsma.xml:49]: check the symbol

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LOC7393Amyatsma.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/LOC7393Amyatsma.xml:8]: I wanted to encode mʸā. I took the fidal character ፙ from wikipedia -> Gəʿəz script. The character looks different in the oxygen and in Github. In wikipedia and as well in github, there is a dash on top, as it is visible in the manuscript. In Oxygen, however it is formed in a hybrid way with a dash on top as well as a dash below as if it is a combination o mʷā ሟ and mʸā ፙ.

Screenshot 2023_Amyatsma

Comment (eu-genia) [reply new/LOC7393Amyatsma.xml:8]: OK, thank you!

**Review by eu-genia — APPROVED** [review]

**Pull request merged**